CONTRIBUTING.md

```diff
@@ -111,6 +111,10 @@ documentation up to date with the markdown documentation present as of commit
 16fd083e78551f866a0cf0c7397542aea1c214d9 and then working on adding the documentation added since
 that commit. 
 
+Since Read the Docs does not currently [support Pipfiles](https://github.com/readthedocs/readthedocs.org/issues/3181) the package requirements are in:
+```
+docs/requirements.txt
```

Comment:
I placed the requirements in the top-level docs directory in the same way that put the PIpfile in the top-level graphql-compiler directory.